packages/history-utility/src/index.ts

```diff
@@ -23,11 +23,14 @@ export type HistoryNode<T> = {
   updatedAt?: Date;
 };
 
+type EmptyWIP = Record<string, never>;
+const EmptyWIP: EmptyWIP = {};
```

Comment:
i still have a problem with the semantics here so how about we use a Symbol and more standard naming convention

```suggestion
const EmptyWIP: EMPTY_WIP = Symbol('empty');
```

Reply:
Sure, I fixed the naming case and used a Symbol.